Transactions.md

```diff
@@ -312,9 +312,57 @@ accept_cet_vbytes = Ceil(accept_cet_weight / 4.0) vbytes
 
 TODO
 
-# References
+# Channel
 
-* [Bitcoin-S implementation](https://github.com/bitcoin-s/bitcoin-s/blob/adaptor-dlc/dlc/src/main/scala/org/bitcoins/dlc/builder)
+## Channel Contract Execution Transaction
+
+The CETs are as described [above](#contract-execution-transaction) with the difference of using their nSequence field to set a relative lock enabling timely use of the punishment path in case of misbehavior.
+
+## Buffer Transaction
+
+In the case of DLC channels, a buffer transaction is inserted between the fund transaction output and the CETs.
+This enables safe revocation of an established contract even before its maturity.
+
+### Input
+
+* version: 2
+* locktime: 0
+* txin count: 1
+  * `txin[0]` outpoint: `txid` of funding transaction and `output_index` 0
+  * `txin[0]` sequence: 0xFFFFFFFE
+  * `txin[0]` script bytes: 0
+  * `txin[0]` witness: `0 <signature_for_pubkey1> <signature_for_pubkey2>`
+
+### Output
+
+* The buffer transaction output script is a P2WSH defined by the following miniscript policy:
+
+```
+or(and(pk(offer_own_pk),pk(accept_own_pk)),or(and(pk(offer_own_pk),and(pk(accept_publish_pk), pk(accept_rev_pk))),and(pk(offer_own_pk),and(pk(offer_publish_pk),pk(offer_rev_pk)))))
```

Comment:
In [maia](https://github.com/comit-network/maia/blob/fc6b78b98407b10b55f8cfd152062ad77f98cd9f/maia/src/protocol/transactions.rs#L277) we have an additional `RefundTransaction` which spends from the `Buffer Transaction` (`CommitTransaction` in our case). This allows you to get your funds back if the other party disappears and the oracle never attests.

Reply:
Yes I have that as well, just didn't mention it. Added a small note l320 let me know if it's clear enough or not.